### ISSUE TITLE
Project Owners 101 Video: Why raise funds in crypto & Giveth?

### ISSUE THREAD

**Issue by ahmadabugosh** [Giveth/giveth-planning#732]

**Comment by laurenluz** [comment]
Perks!
- on the blockchain, transparency of funds
- zero fees to add project or get funds, nothing skimmed off the top
- zero intermediary, P2P... can't be taken away from you/withheld
- sovereignty... you are responsible for your keys & your funds
- DAO decision making on the platform... you have a say in what gets developed
- DeFi & earning opportunities
- global market... funds can be sent all around the world

GIVbacks
- donors get rewards
- like tax deductible donations w/o governments or borders

Moving to DAOs.. feedback loops b/w donors & projects

@ahmadabugosh @WhyldWanderer think I'm missing anything?

**Comment by ahmadabugosh** [comment]
Looks good! Maybe add "exposure to crypto market donors (another donation channel) that might not have been exposed to your project otherwise". Basically, how it's another marketing channel for them in the crypto world (a market where people generally have funds).

**Comment by ahmadabugosh** [comment]
Hey @laurenluz when you're back in action, can you take a crack at recording this one? 🙏

**Comment by ahmadabugosh** [comment]
Hey @laurenluz thanks for recording this! I think it's a really good comprehensive overview. I did some basic editing and posted it, let me know if you're ok with taking it live: https://www.youtube.com/watch?v=c7q3UDIzniU

**Comment by laurenluz** [comment]
is there anyway that you could cut a little back and forth between my screen share & my face? it looks a little funny to see my tiny face in the corner covered up by the giveth logo

![image](https://user-images.githubusercontent.com/75490971/197244474-09c8443b-a9da-4a59-b440-6addd58d7e28.png)

**Comment by laurenluz** [comment]
if not. I think it's totally fine!!

**Comment by ahmadabugosh** [comment]
hey @laurenluz I don't think I can switch back and forth easily (since it looks like the file was recorded in one video track), but I did manage to make you bigger, and also made the background dim until you started showing something on your screen https://youtu.be/rAzsY8__lIU

**Comment by laurenluz** [comment]
I think the dim background makes the video look kind of dark and unpleasant... and it's like that for so long at the beginning... maybe keep it bright? sorry for the nitpicking, I'm also just open to you leaving it as is. Up to your judgement.

**Comment by ahmadabugosh** [comment]
@laurenluz All good that's why we have this review process :) 

 Yeah I think making it lighter is a brighter video in general (let me know if you think it's better): https://www.youtube.com/watch?v=28zEiy9PQYc

**Comment by laurenluz** [comment]
I like that better. last thought... when I start actually doing the screen sharing, it feels a little funny to have some of it cut off. especially on this page. can you make me smaller when the screen movement starts? otherwise if feels a bit frustrating to not be able to see well.

![image](https://user-images.githubusercontent.com/75490971/197270568-e4525d48-87af-469e-9c61-ee8fe3dc0167.png)

**Comment by ahmadabugosh** [comment]
@laurenluz this better? https://www.youtube.com/watch?v=u5nUpYIbABE

**Comment by laurenluz** [comment]
yeah! let's call it done

**Comment by danelsuga** [comment]
I noticed the new video is not part of the series. Is that OK for now?
new one: 
https://www.youtube.com/watch?v=u5nUpYIbABE

old one:
https://www.youtube.com/watch?v=u5nUpYIbABE&list=PL4Artm1rmCWFi-qEkOtjl9nL4tojSIIKm&index=4

This may be a non-issue that I don't understand! Thanks.

**Comment by ahmadabugosh** [comment]
Hey @danelsuga they look like the same video to me 😅

(look at the part before the "&list" in the url)

![tumblr_dc923c1b6f1f6e38fcea9beda25a9960_82270412_540](https://user-images.githubusercontent.com/4855928/198591021-59dadbbf-8a89-4fae-a178-f4ed5e60002a.gif)

**Comment by danelsuga** [comment]
🤣

**Comment by danelsuga** [comment]
So, when I watch this video, Lauren's face is covered up by the Giveth logo, unless I keep my play bar (blanking on the name for that) visible. Pics to show. 
![image](https://user-images.githubusercontent.com/81351744/198890689-2160c96c-c922-43a7-9104-f8ae5c560e15.png)
![image](https://user-images.githubusercontent.com/81351744/198890707-691a2565-3222-4c86-90fc-bcae89af0329.png)

**Comment by ahmadabugosh** [comment]
Thanks @danelsuga! That's actually a YouTube channel feature that displays a watermark on our videos. I changed it so it doesn't show the entire video, and now it should just show towards the end (I tested it out and now it just shows up in the last 15 seconds of the video when watching it for the first time).

<img width="657" alt="Screen Shot 2022-10-30 at 11 05 04 PM" src="https://user-images.githubusercontent.com/4855928/198896843-aed216db-09c6-4f09-8528-12e66d59dc82.png">
